### DISCUSSION TITLE
Position (geofenced) is not trasmitted in time

### DISCUSSION THREAD

**Discussion by Klagio** [teslamate-org/teslamate#1388] [Q&A]
Hi,

I use the geofence position with Home Assistant, when I approach the car to home it fires an automation in Home assistant (opening garage turn on driveway lights and so on). It worked for months and until last week was working great.

Recently the position it is not updated: when I go to see the device_tracker in Home Assistant it shows a position of few hours before. So it gets updated but of course not in time for my automation to fire.

Not sure how to troubleshoot, but my settings is the same as when it was perfectly working and for months. I updated to latest Teslamate firmware as of today.

**Comment by LelandSindt** [comment]
when trying to observe and troubleshoot mqtt  values I have found `mosquitto_pub` and `mosquitto_sub` to be very helpful.

you could subscribe and publish  to `teslamate/cars/1/geofence`  to test your Home Assistant connection/automation.
